src/otx/algo/instance_segmentation/heads/maskdino_encoder.py

```diff
@@ -6,26 +6,36 @@
 
 from __future__ import annotations
 
+from typing import Any, Callable, ClassVar
+
 import numpy as np
 import torch
 from torch import Tensor, nn
-from torch.cuda.amp import autocast
 from torch.nn import functional as f
 from torch.nn.init import normal_
 
-from otx.algo.detection.heads.rtdetr_decoder import MSDeformableAttention as MSDeformAttn
-from otx.algo.instance_segmentation.heads.pixel_decoder.position_encoding import PositionEmbeddingSine
-from otx.algo.instance_segmentation.layers.batch_norm import get_norm
+from otx.algo.common.layers.position_embed import PositionEmbeddingSine
+from otx.algo.common.layers.transformer_layers import MSDeformableAttention, VisualEncoder, VisualEncoderLayer
 from otx.algo.instance_segmentation.utils.utils import (
-    Conv2d,
     ShapeSpec,
     c2_xavier_fill,
-    get_clones,
 )
+from otx.algo.modules.conv_module import Conv2dWithNormActivation
 
 
 class MSDeformAttnTransformerEncoderOnly(nn.Module):
```

Comment:
Why this module is called "Only"?

Reply:
I don't know. It's from the original MaskDINO code, but I can rename it.